src/foam/u2/texteditor/Popup.js

```diff
@@ -0,0 +1,69 @@
+foam.CLASS({
+  package: 'foam.u2.texteditor',
+  name: 'Popup',
+  extends: 'foam.u2.View',
+
+  css: `
+  ^menu {
+    position: absolute;
+    background-color: #f0f0f0;
+    min-width: 50px;
+    min-height: 50px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+  }
+
+  ^menu  button {
+    display: block !important;
+    width: 100px !important;
+    height: 40px;
+    text-align: left !important;
+    padding: 16px;
+    border-radius: 0px;
+  }
+
+  .foam-u2-DetailView-toolbar {
+    display: block;
+    padding-top: 8px;
+  }
+  `,
+
+  properties: [
+    {
+      name: 'button'
+    },
+    {
+      class: 'foam.u2.ViewSpec',
+      name: 'view'
+    },
+    {
+      class: 'Boolean',
+      name: 'hidden',
+      value: true
+    }
+  ],
+  
+  methods: [
+    function initE() {
+      var self = this;
+      self.SUPER();
+      self
+          .start()
+            .add(self.button$)
+            .on('click', function(e1) {
+              self.hidden = ! self.hidden;
+              if ( ! self.hidden ) {
```

Comment:
Are you ever going to unhide the popup or is it now hidden forever? If forever, just call self.remove() to remove it from the DOM. Then there's no need to remove the event listener.

Reply:
@kgrgreer Yes plan is to be able to hide and unhide the popup.